From eec582eb19880c64ffd28ceecd11de80fc218e0d Mon Sep 17 00:00:00 2001
From: Matt Fleming <matt.fleming@intel.com>
Date: Tue, 21 Apr 2015 16:50:31 +0100
Subject: [PATCH 6/6] drivers/linux: Don't build userland app automatically

The OpenEmbedded build infrastructure has very different toolchain
settings for compiling kernel modules and userland utilities.

Don't automatically build the userland utility and instead expect the
user to do that manually.

Signed-off-by: Matt Fleming <matt.fleming@intel.com>
---
 source/drivers/linux/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/drivers/linux/Makefile b/source/drivers/linux/Makefile
index 7eefc78..5e7bcce 100644
--- a/source/drivers/linux/Makefile
+++ b/source/drivers/linux/Makefile
@@ -9,7 +9,7 @@ all:	clean chipsec$(VERSION)
 
 chipsec$(VERSION) : clean
 	make -C $(KERNEL_SRC_DIR) SUBDIRS=`pwd` modules
-	cd ../../tool/chipsec/helper/linux; make
+#	cd ../../tool/chipsec/helper/linux; make
 
 install:
 	./run.sh
-- 
2.1.0

